bindings/nostr-nodejs/src/nips/nip26.rs

```diff
@@ -11,6 +11,40 @@ use nostr::secp256k1::schnorr::Signature;
 use crate::error::into_err;
 use crate::{JsKeys, JsPublicKey};
 
+/// Create a NIP-26 delegation tag (including the signature).
+/// See also validate_delegation_tag().
+#[napi]
+pub fn create_delegation_tag(
+    delegator_keys: &JsKeys,
+    delegatee_pubkey: &JsPublicKey,
+    conditions: String,
+) -> Result<String> {
+    match nip26::create_delegation_tag(delegator_keys.deref(), delegatee_pubkey.into(), &conditions)
+    {
+        Ok(tag) => Ok(tag.to_string()),
+        Err(_) => Ok("".to_string()),
+    }
```

Comment:
Replace this with:

```rust
let tag = nip26::create_delegation_tag(delegator_keys.deref(), delegatee_pubkey.into(), &conditions).map_err(into_err)?;
Ok(tag.to_string())
```

Reply:
I could not get this done this way. The only way I could do is using `format!("{:?}, tag)` (and adding Debug to few types), but that is using dervied Debug impl, which formats json differently.
For some reason I could not get .to_string() to be applied on the result.

Reply:
Add `.map_err(into_err)?`

Reply:
To me, the solution that I have written, works.

Reply:
Copy and paste what I have written. Maybe the line is too long and you missed a piece of code.

Reply:
Oh yes, I missed the map_err part! OK now